varats-core/varats/provider/patch/patch_provider.py

```diff
@@ -237,12 +238,10 @@ class PatchProvider(Provider):
     def __init__(self, project: tp.Type[Project]):
         super().__init__(project)
 
-        # BB only performs a fetch so our repo might be out of date
-        pull_current_branch(self._get_patches_repository_path())
+        self._update_local_patches_repo()
+        repo_path = self._get_patches_repository_path()
 
-        patches_project_dir = Path(
-            self._get_patches_repository_path() / self.project.NAME
-        )
+        patches_project_dir = Path(repo_path / self.project.NAME)
```

Comment:
```suggestion
        patches_project_dir = repo_path / self.project.NAME
```
repo_path is already a path, so if NAME is a string you can directly add it to the path and get a path